pkgs/build-support/setup-hooks/role.bash

```diff
@@ -17,7 +17,7 @@ function getRole() {
             role_post='_FOR_TARGET'
             ;;
         *)
-            echo "@name@: used as improper sort of dependency" >2
+            echo "@name@: used as improper sort of dependency" >&2
```

Comment:
both changes to this file were fixed in https://github.com/nixos/nixpkgs/commit/8ed1104a4a0c7f6cc289dd35b5dfd46539dcd8be